### ISSUE TITLE
Cardboard with AR Core in Unity

### ISSUE THREAD

**Issue by cutiemod** [googlevr/cardboard#256]
Cardboard VR mode doesn't initialize correctly if I have ARCore in my project. 

![image](https://user-images.githubusercontent.com/48511696/116791916-266f3180-aac6-11eb-9765-b231c528dcca.png)

I'm using the "VrMode" example scene and it goes through initialization seemingly fine (no errors) but the screen stays a single image from a single camera. Like this:

![image](https://user-images.githubusercontent.com/48511696/116792139-831f1c00-aac7-11eb-847e-38d74a19d605.png)

If I untick the ARCore box it works as expected. Is that an issue on Unity's end or am I missing something? Googling around I found people combining ARCore and Cardboard in a single app so it must be possible, but I couldn't find any similar issues to mine. 

I'm using Unity 2020.3.2f1 LTS, AR Core XR plugin 4.0.12 and Google Cardboard XR 1.4.1.

**Comment by AndrewSpalato-FGMNT** [comment]
You'll need to start and stop the individual XR subsytems manually. ARCore seems to be initializing and active first in your configuration

[.](https://forum.unity.com/threads/xr-plugin-management-toggling-vr-on-and-off-doesnt-work-right.1022275/)

**Comment by cutiemod** [comment]
Thanks, that works. They actually have a page about it which I somehow missed, my bad: https://docs.unity3d.com/Packages/com.unity.xr.management@4.0/manual/EndUser.html

**Comment by GabrielR** [comment]
@cutiemod Did you get this working with your own loader? ARFoundation has really poor documentation.  Any chance you could share a code sample?

**Comment by cutiemod** [comment]
@GabrielR I did not, I don't remember exactly why though. I think my app was crashing on either AR or VR initializing, and I couldn't pinpoint why (probably something else in the project conflicting with Cardboard), so I just gave up on combining both in one app since that wasn't a priority. I did manage to get it "working" in editor (can't properly test AR there) though, using google cardboard enable/disable VR mode example from their unity package.
